### ISSUE TITLE
Translatable inputs and textareas should communicate that they are required

### ISSUE THREAD

**Issue by greatislander** [accessibility-in-action/platform#1155]
**Is your feature request related to a problem? Please describe.**

One should be able to specify that a translablte input or textarea is required, either in one or multiple langauges.

**Describe the solution you'd like**

A `required` attribute on the component should set the `required` attribute on the default locale if no value is provided. It should also accept an array of language codes and should set the `required` attributes on each language if the language is present in the translations array.

**Describe alternatives you've considered**
Not applicable.

**Additional context**
Not applicable.

**Comment by jobara** [comment]
@greatislander I wonder if instead it should set the required attribute on all fields by default. Also perhaps the default value should always be required if the required attribute is provided, with or without an array. Thoughts?

**Comment by jobara** [comment]
Depending on how https://github.com/whatwg/html/issues/6868 is resolved, we may someday be able to apply required to a fieldset instead.

**Comment by jobara** [comment]
@greatislander I am working through this and noticed that the [text of the button](https://github.com/accessibility-exchange/platform/blob/76cf21279a10d641c9f6a6f3c9206a0c599a332e/resources/views/components/translatable-textarea.blade.php#L26) to add the additional language is derived from the label of the first field. The `(required)` text indicator is included in the label.

I had gotten the array of required locales working, but am realizing that the button text will still say required even though the required property may not be included. Because it is part of the label there isn't a way for it be easily removed programatically.

So the option would be to change the functionality here to make all the fields required or not at the same time, or modify how the required text is added to the labels and button text. I think given the use case, I'm inclined to do the former, but would like to hear your thoughts first.

**Comment by greatislander** [comment]
@jobara I think we need to do the latter, because we can't make both fields required. That would confuse user expectations since you'll still be able to save the field if only one language is provided.

**Comment by jobara** [comment]
I guess the problem is that there doesn't actually appear to be an adequate way of describing conditionally required fields. In this case it's required to have one of the fields filled out but none of the fields are specifically required. So if you have one field, and mark the initial one as required it's accurate, but as soon as you add the additional field you run into the issue.

**Comment by jobara** [comment]
@greatislander and I discussed this issue. The current implementation appears to be marking the initial field as `required` or not as indicated to the component. However, we don't want to mark all of the fields as required. So we'll leave the additional fields as not marked required and make use of the `shortLabel` attributes on the translatable components to specify a label for the additional fields that doesn't include the `(required)` text.
